snakemake/__init__.py

```diff
@@ -2288,6 +2295,11 @@ def adjust_path(f):
             args.jobscript = adjust_path(args.jobscript)
         if args.cluster:
             args.cluster = adjust_path(args.cluster)
+        if args.cluster_config:
+            if isinstance(args.cluster_config, list):
+                args.cluster_config = [adjust_path(cfg) for cfg in args.cluster_config]
+            else:
+                args.cluster_config = adjust_path(args.cluster_config)
         if args.cluster_sync:
```

Comment:
This enables a cluster config file to be placed in the profile directory, to configure e.g. a project or user code for a given cluster.